src/test/isolation2/input/resgroup/resgroup_move_query.source

```diff
@@ -135,7 +139,7 @@ SELECT num_running FROM gp_toolkit.gp_resgroup_status WHERE rsgname='rg_move_que
 1&: SELECT pg_sleep(3);
 2: SET ROLE role_move_query_mem_small;
 2: BEGIN;
-2: SELECT hold_memory_by_percent_on_qe(1,0.1);
+2: SELECT hold_memory_by_percent_on_qe(1,0.2);
```

Comment:
Why 0.1 is replaced with 0.2 here and below?

Reply:
reverted